.github/workflows/release-next.yml

```
@@ -1 +1,59 @@
+name: Release Next"
 
+on:
+  workflow_dispatch:
+    inputs:
+      ui_bundle_url:
+        description: "ui_bundle_url"
+        required: false
+        default: ""
+
+permissions:
+  contents: write
+  packages: write
+
+env:
+  # change this to embed a different UI dashboard
+  UI_BUNDLE_URL: https://github.com/rancher/dashboard/releases/download/epinio-standalone-v1.2.0.0.0.1/rancher-dashboard-epinio-standalone-embed.tar.gz
+
+jobs:
+  release:
+    runs-on: self-hosted
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+          fetch-depth: 0
+      -
+        name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: '1.18'
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      -
+        name: Login to GitHub Docker Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      -
+        name: Download dashboard
+        run: |
+          mkdir ui
+          wget "${{ env.UI_BUNDLE_URL }}"
+          tar xfz *.tar.gz -C ui
+      -
+        name: Get current tag
+        id: get_tag
+        run: echo ::set-output name=TAG::${GITHUB_REF/refs\/tags\//}
```

`::set-output` is deprecated please replace by:
```suggestion
        run: echo "TAG=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_OUTPUT
```
Anyway, I don't see where `get_tag` is used so not sure if we really need it.

---

Ah yes, it was a copy-paste from the other action, but here it's not used. I'll fix it in the other one. :+1: